lib/cc/engine/analyzers/ruby/main.rb

```diff
@@ -24,6 +24,10 @@ class Main < CC::Engine::Analyzers::Base
 
           def process_file(file)
             RubyParser.new.process(File.binread(file), file, TIMEOUT)
+          rescue Timeout::Error
```

Comment:
> if the other languages don't raise, then it shouldn't be pushed up.

I believe they do, though: the other languages use `CommandLineRunner`, which uses `Timeout`, and I don't think that's already caught by those.